decompilation/MohoEngine.dll.md

```
@@ -0,0 +1,738 @@
+    int            speedFlags,  // 0=normal, 1=adjustable, 4=fast
+    bool           isHost
+);
+```
+
+---
+
+## 3. Sim::AdvanceBeat – Full Tick Pipeline (Decompiled, 0x10318F50)
+
+This is the **core simulation tick function** (299 lines decompiled).
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)